modules/bitgo/test/v2/unit/coins/sol.ts

```diff
@@ -903,14 +903,25 @@ describe('SOL:', function () {
     });
 
     it('should rebuild tx request for hot wallets', async () => {
-      const rebuiltTx = {
+      const rebuiltTx: TxRequest = {
         txRequestId,
         unsignedTxs: [{
           serializedTxHex: 'deadbeef',
           signableHex: 'serializedTxHex',
           derivationPath: 'm/0',
         }],
         transactions: [],
+        date: new Date(),
```

Comment:
just wondering, doesn't the tx request comes over a rest api? -- will it ever be type `Date` ? 🤔 should always be string no?